etc/environment-tests.yml

```diff
@@ -0,0 +1,11 @@
+name: pySDC
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - pytest
+  - pytest-benchmark
+  - pytest-timeout
+  - pytest-order
+  - coverage[toml]
+  - sphinx
```

Comment:
Does it make sene to do a special environment for post processing such we don't need to install `sphinx` in every environment? I don't know how long this takes. But if we can shave a few seconds off every single pipeline, that might be worth doing.